nservicebus/hosting/aws-lambda-simple-queue-service_delayed-delivery_sqslambda_[1,).partial.md

```diff
@@ -0,0 +1,3 @@
+[Delayed retries](/nservicebus/recoverability/configure-delayed-retries.md) is disabled by default when using AWS Lambda. Delayed retries cam be enabled as follows:
```

Comment:
it says "delayed retries ARE disabled" in the version above but "delayed retries IS disabled" in this.